client.go

```diff
@@ -97,6 +97,7 @@ const (
 type Client interface {
 	Party
 	Start()
+	Stop()
```

Comment:
Please document stop the doc comment of the interface type

Reply:
done. Out of curiosity: Why aren't those docs above the methods itself? Just noticed now that they are on the interface doc rather than on the methods itself.